spec/controllers/course/controller_spec.rb

```diff
@@ -64,5 +50,28 @@ def publicly_accessible?
         end
       end
     end
+
+    describe '#current_component_host' do
+      it 'returns the current course' do
+        allow(controller).to receive(:current_course).and_return(course)
```

Comment:
I'm trying to isolate it from other tests here, make it not depend on <code>current_course</code>